### PR TITLE
Update Nav After Login

### PR DESCRIPTION
**Pull request by t-schell**

### CODE REVIEW

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -6,11 +6,13 @@ import '../../css/NavBar/NavBar.css'
 
 export default class NavBar extends React.Component {
   render() {
+    const loggedInUser = this.props.loggedInUser
```
Comment (mchuong1) [inline src/components/NavBar/NavBar.js]: Can use destructuring here

File: src/components/NavBar/NavLeft.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
 import React from "react";
 import { Link } from 'react-router-dom'
 
-export default class NavLeft extends React.Component {
-  render(){
+export default function NavLeft(props) {
+  
+    const loggedIn = props.loggedIn
```
Comment (mchuong1) [inline src/components/NavBar/NavLeft.js]: same here

**Comment by mchuong1** [comment]
image
Navbar stays black after searching.
It should change to green and only be black on landing page
This happens when the user is not logged in.

**Comment by mchuong1** [comment]
image
When logging out. The side menu should close as well.

File: src/components/NavBar/NavBar.js (hunk 1 above)
Comment (t-schell) [reply src/components/NavBar/NavBar.js]: So what's the benefit of using destructing when there's only one item to pull out?

**Comment by t-schell** [comment]
> image
> When logging out. The side menu should close as well.

**Comment by t-schell** [comment]
Fixed the last two comments. Waiting for your thoughts on first two

**Comment by t-schell** [comment]
Upon further testing navbar bit is still wonky. Working on it now

**Comment by t-schell** [comment]
> Upon further testing navbar bit is still wonky. Working on it now

Fixed it

File: src/components/NavBar/NavBar.js (hunk 1 above)
Comment (mchuong1) [reply src/components/NavBar/NavBar.js]: just makes the code scalable in case we want more than one variable in the future.

**Comment by mchuong1** [comment]
image
We can talk about the first item.
Second is that the home logo isnt changing color when searching not logged in
Third, i dont think your master is up to date so you will need to merge from master again

**Comment by t-schell** [comment]
Changes included in new PR. Closing...

**Pull request closed without merging**